org.civicrm.reqangular/src/common/services/angularDate/date-format.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'common/angular',
+    'common/modules/angularDate'
+], function (angular, module) {
+    'use strict';
+
+    /**
+     * This service provides access to date format from CRM settings
+     */
+    module.factory('DateFormatService', ['$q', function ($q) {
```

Comment:
I think the naming convention for services is not to put `Service` as a prefix for the name